peer/brontide.go

```diff
@@ -2286,25 +2283,30 @@ func (p *Brontide) queue(priority bool, msg lnwire.Message,
 // ChannelSnapshots returns a slice of channel snapshots detailing all
 // currently active channels maintained with the remote peer.
 func (p *Brontide) ChannelSnapshots() []*channeldb.ChannelSnapshot {
-	p.activeChanMtx.RLock()
-	defer p.activeChanMtx.RUnlock()
+	snapshots := make(
+		[]*channeldb.ChannelSnapshot, 0, p.activeChannels.Len(),
+	)
+
+	p.activeChannels.ForEach(func(_ lnwire.ChannelID,
```

Comment:
Nit: I find it mildly confusing that sometimes `Range` is used and sometimes `ForEach` is used, with no apparent pattern.